akka-persistence-testkit/src/main/scala/akka/persistence/testkit/scaladsl/PersistenceTestKit.scala

```diff
@@ -424,9 +423,9 @@ object SnapshotTestKit {
  */
 @ApiMayChange
 class PersistenceTestKit(system: ActorSystem)
-    extends PersistenceTestKitOps[PersistentRepr, JournalOperation]
-    with ExpectOps[PersistentRepr]
-    with HasStorage[JournalOperation, PersistentRepr] {
+    extends PersistenceTestKitOps[(PersistentRepr, Metadata), JournalOperation]
```

Comment:
corresponding changes in javadsl not needed?

Reply:
These type params only change internal things, the ops classes are private[testkit]